ui/urls.py

```diff
@@ -31,10 +32,20 @@
 
 
 def get_sqs():
```

Comment:
I think it could be trivially moved into the search app. I'll take a run at it.

I can add a test to `ui/tests/test_learningresources_views.py` which should fully cover this function.